modules/standard/FileSystem.chpl

```diff
@@ -644,7 +644,7 @@ iter findfiles(startdir: string = ".", recursive: bool = false,
                hidden: bool = false, param tag: iterKind): string
        where tag == iterKind.standalone {
   if (recursive) then
-    forall subdir in walkdirs(startdir, hidden=hidden) do
+    forall subdir in walkdirs(startdir, hidden=hidden) with (ref hidden) do
```

Comment:
This change is curious. Can you say more about what happened here? Is it still necessary? Is it a workaround?